File: src/URI/URI_1013.java
package URI;
import java.util.Scanner;

public class URI_1013 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int a, b, c, maior;		
		
		a = sc.nextInt();
		b = sc.nextInt();
		c = sc.nextInt();
		
		maior = (a + b + Math.abs(a - b)) / 2;
		maior = (c + maior + Math.abs(c - maior)) / 2;
		
		System.out.println(maior + " eh o maior");
		
		sc.close();

	}

}

File: kotlin/URI_1013.kt
import java.util.*
import kotlin.math.abs

fun main(args: Array<String>) {
    Locale.setDefault(Locale.US)
    val sc = Scanner(System.`in`)
    val a = sc.nextInt()
    val b = sc.nextInt()
    val c = sc.nextInt()

    val maiorAB = (a + b + abs(a - b)) / 2
    val maior = (maiorAB + c + abs(maiorAB - c)) / 2

    println("$maior eh o maior")
}